lib/gooddata/models/project.rb

```diff
@@ -264,6 +264,9 @@ def transfer_processes(from_project, to_project, options = {})
           to_process = if process.path
                          to_process.delete if to_process
                          GoodData::Process.deploy_from_appstore(process.path, name: process.name, client: to_project.client, project: to_project)
+                       elsif process.component
```

Comment:
Refers to 'process' more than self (maybe move it to another class?) (FeatureEnvy)